packages/extension-ui/src/components/Address.tsx

```diff
@@ -504,9 +506,13 @@ export default styled(Address)(
   }
 
   .fullAddress {
+    display: flex;
+    align-items: center;
+    column-gap: 4px;
+
     overflow: hidden;
     text-overflow: ellipsis;
-    max-width: 162px;
+    max-width: 178px;
```

Comment:
Sorry for being to lazy to run it on my own, but won't that break some views?

Reply:
I couldn't find any